kedro/extras/transformers/README.md

```diff
@@ -0,0 +1,6 @@
+# Information regarding transformers in Kedro

```

Comment:
I don't think the title needs to be this long, it can probably just be "transformers"